numberpad.py

```diff
@@ -476,13 +476,33 @@ def load_keymap_listener_x11():
       log.exception("X11 load keymap listener error. Exiting")
       os.kill(os.getpid(), signal.SIGUSR1)
 
+def get_libinput_touchpad_event_number() -> int:
```

Comment:
@sleddev Why you not used the variable `touchpad`, origin code:

```
# Start monitoring the touchpad
fd_t = open('/dev/input/event' + str(touchpad), 'rb')
d_t = Device(fd_t)
```

It is not the same?